File: app/src/main/java/no/nav/sosialhjelp/soknad/consumer/bostotte/dto/BostotteMottaker.java
package no.nav.sosialhjelp.soknad.consumer.bostotte.dto;


import no.nav.sbl.soknadsosialhjelp.soknad.okonomi.opplysning.JsonOkonomiOpplysningUtbetaling;

public enum BostotteMottaker {
    KOMMUNE(JsonOkonomiOpplysningUtbetaling.Mottaker.KOMMUNE.value()), HUSSTAND(JsonOkonomiOpplysningUtbetaling.Mottaker.HUSSTAND.value());

    private final String value;

    BostotteMottaker(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/client/husbanken/enums/BostotteMottaker.kt
package no.nav.sosialhjelp.soknad.client.husbanken.enums

import no.nav.sbl.soknadsosialhjelp.soknad.okonomi.opplysning.JsonOkonomiOpplysningUtbetaling

enum class BostotteMottaker(val value: String) {
    KOMMUNE(JsonOkonomiOpplysningUtbetaling.Mottaker.KOMMUNE.value()),
    HUSSTAND(JsonOkonomiOpplysningUtbetaling.Mottaker.HUSSTAND.value());
}
